tests/integration/minibroker_test.go

```diff
@@ -142,7 +154,7 @@ var _ = Describe("classes", func() {
 					"Command": []string{
 						"sh", "-c",
 						"mongo" +
-						" ${DATABASE_URL}",
+							" ${DATABASE_URL}",
```

Comment:
```suggestion
						"mongo ${DATABASE_URL}",
```

Was the old code even correct? Why was `connectionStatus` a completely different argument to `sh` (not `mongo`)‽

Reply:
Interesting. Good catch.

Reply:
It was not correct... I missed it in my original review of the PR that introduced this code. Fixed with 6600dd0.